README.md

```diff
@@ -47,3 +47,17 @@ Chrome OS: ChromeVox with Chrome.
 ## Testing Supported By
 
 <a href="https://www.browserstack.com/"><img width="160" src="https://user-images.githubusercontent.com/594745/69711802-fc138a80-1101-11ea-9b30-3e90c274737a.png" alt="BrowserStack"/></a>
+
```

Comment:
Thanks for the addition. This kind of information we currently have in DEVELOPER.md where there is already such a section. So I suggest to add your changes there for the moment.

https://github.com/sbb-design-systems/sbb-angular/blob/main/DEVELOPER.md

Reply:
Thanks for the input ! I wasn't aware anymore of that DEVELOPER.md file. I moved my text over there.